Microsoft.HealthVault.Fhir/Transformers/ObservationToBloodPressure.cs

```diff
@@ -0,0 +1,100 @@
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+using System;
+using Hl7.Fhir.Model;
+using Microsoft.HealthVault.Fhir.Codes.HealthVault;
+using Microsoft.HealthVault.Fhir.Constants;
+using Microsoft.HealthVault.ItemTypes;
+
+namespace Microsoft.HealthVault.Fhir.Transformers
+{
+    internal static class ObservationToBloodPressure
+    {
+        internal static BloodPressure ToBloodPressure(this Observation observation)
+        {
+            var bloodPressure = observation.ToThingBase<BloodPressure>();
+            
+            bloodPressure.When = ObservationToHealthVault.GetHealthVaultTimeFromEffectiveDate(observation.Effective);
+
+            bloodPressure.IrregularHeartbeatDetected = observation.GetBoolExtension(HealthVaultVocabularies.IrregularHeartBeatExtensionName);
+
+            if(observation.Component != null)
+            {
+                foreach(var component in observation.Component)
+                {
+                    if(component.Code != null && component.Code.Coding != null)
+                    {
+                        foreach (var code in component.Code.Coding)
+                        {
+                            if (code.Code != null)
+                            {
+                                if (code.System == HealthVaultVitalStatisticsCodes.System)
+                                {
+                                    if (code.Code == HealthVaultVitalStatisticsCodes.BloodPressureDiastolic.Code)
+                                    {
+                                        SetDiastolic(bloodPressure, component);
+                                    }
+                                    else if (code.Code == HealthVaultVitalStatisticsCodes.BloodPressureSystolic.Code)
+                                    {
+                                        SetSystolic(bloodPressure, component);
+                                    }
+                                    /*else if (code.Code == HealthVaultVitalStatisticsCodes.HeartRate.Code)
```

Comment:
Is there any reason for this to be commented out instead of either implemented or removed?